h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -111,6 +111,7 @@ public void computeCrossValidation() {
    */
   @Override
   public void cv_computeAndSetOptimalParameters(ModelBuilder[] cvModelBuilders) {
+    if(_parms._max_runtime_secs != 0) _parms._max_runtime_secs = 0;
     if(_parms._lambda_search) {
```

Comment:
I thought setting _parms._max_runtime_secs to 0 will disable maximum run time restriction?